packages/components/src/FormatTime/FormatTime.mdx

```diff
@@ -0,0 +1,45 @@
+---
+name: FormatTime
+menu: Components
+route: /components/time-formatter
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { CivilTime } from "@std-proposal/temporal";
+import { FormatTime } from ".";
+
+# FormatTime
+
+<ComponentStatus stage="ready" responsive="yes" accessible="yes" />
+
+In Jobber a FormatTime is used to ensure that time is displayed in the expected
+format. No text styling is applied, this simply formats the text.
+
+## Simple Example
+
+```ts
+import { FormatTime } from "@jobber/components/FormatTime";
+```
+
+<Playground>
+  <FormatTime time={new CivilTime(2, 35)} />
+  -
+  <FormatTime time={new CivilTime(18, 35)} />
+</Playground>
+
+## Specify Format
+
+<Playground>
+  <FormatTime time={new CivilTime(2, 35)} use24HourClock={true} />
+  -
+  <FormatTime time={new CivilTime(18, 35)} use24HourClock={true} />
+  <br />
+  <FormatTime time={new CivilTime(2, 35)} use24HourClock={false} />
+  -
+  <FormatTime time={new CivilTime(18, 35)} use24HourClock={false} />
+</Playground>
+
+## FormatTime Properties
+
+<Props of={FormatTime} />
```

Comment:
Just two comments, 
1.) This `Props of`, is not actually listing anything, this section is empty when I view it (I re-ran npm start) and refreshed. 
2.) I think the package-lock changes were from me, I am not sure if they are needed?? Should we remove them before merging?

Reply:
1. They are showing here: https://deploy-preview-87--jobber-atlantis.netlify.com/components/time-formatter

![image](https://user-images.githubusercontent.com/131327/63597285-4e309700-c57a-11e9-9cd3-c06f5e3eaa73.png)

Sometimes when running in dev mode on your local machine you need to do an `npm run clean` to clear some caching. The Netlify branch previews are usually an easier way to test things though. 😄 

2. The lock file should be fine.